server/services/overviewPageService.ts

```diff
@@ -162,7 +161,7 @@ export default class OverviewPageService {
       offencesOverview,
       prisonName: prisonerData.prisonName,
       staffRoles: staffRoles.map(role => role.role),
-      alertsSummary: this.getAlertsSummary(inmateDetail, nonAssociations, userCaseLoads),
```

Comment:
The user caseloads where used to determine whether to show or nor the link to non-associations. This doesn't happen anymore so this parameter/argument is not necessary anymore.